index.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;0,700;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Luiz Froes</title>
+  </head>
+
+  <body>
+    <!--This is the header with the nav bar-->
+    <header class="header">
+      <!--This is the Navigation bar with the portfolio website links-->
```

Comment:
Again, your comments in this file can be much more brief too. Eg `<!--Navigation bar-->`

Reply:
All done!!